action.yml

```diff
@@ -55,7 +55,10 @@ runs:
     - name: Run maven
       run: |-
         test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
-        mvn -B package ${SKIP_TESTS}
+      mvn -B ${{ maven-additional-options:
```

Comment:
Are you sure that you can inline such an option? The option should be added at the top (line 3), where the other inputs are defined.

Additionally the three dots "...", where just a placeholder for the actual goals. So the mvn call at this place should look like this:
```
mvn -B ${{ maven-additional-options }} package ${SKIP_TESTS}
```

Reply:
have to agree with @gclaussn .
The option itself is already defined in the file.
```
maven-release-options:
    description: Any extra Maven options for the release process (-DskipTests is already implied)
    required: false
    default: ""
```

I'd go as far as including the `inputs` -->
mvn -B ${{ inputs.maven-release-options }} package ${SKIP_TESTS}